src/components/editor/Bindings/Selector.tsx

```diff
@@ -248,7 +248,7 @@ function BindingSelector({
                 </Stack>
             </Box>
 
-            <Box sx={{ height: 280 }}>
+            <Box sx={{ height: 480 }}>
```

Comment:
Just thinking... we should probably end up with a system/pattern for heights the same way we do for zindex.